pkg/restore/pod_action.go

```diff
@@ -85,6 +86,11 @@ func (a *PodAction) Execute(input *velero.RestoreItemActionExecuteInput) (*veler
 	if err != nil {
 		return nil, errors.WithStack(err)
 	}
-
-	return velero.NewRestoreItemActionExecuteOutput(&unstructured.Unstructured{Object: res}), nil
+	restoreExecuteOutput := velero.NewRestoreItemActionExecuteOutput(&unstructured.Unstructured{Object: res})
+	if pod.Spec.PriorityClassName != "" {
+		a.logger.Infof("Adding priorityclass %s to AdditionalItems", pod.Spec.PriorityClassName)
```

Comment:
```suggestion
                a.logger.Infof("Restoring priorityclass %s from AdditionalItems", pod.Spec.PriorityClassName)
```

Reply:
Not sure I agree with this feedback.  At this step, we only add it to the AdditionalItems to be restored.  Whether it will be restored or not depend on more configuration setting which will be done in the restoreItem.  The restoreItem itself already have the log for "Restoring ..."

Reply:
I think I agree with @phuongatemc here. The specific action is adding to Additional items. The restoring from additional items happens in the func that calls this one after the RestoreItemAction returns.

Reply:
Sounds good, thanks for the explanation.